app/components/rest/clusters.ts

```diff
@@ -61,11 +61,8 @@ export class ClusterService {
     constructor(rest: restangular.ICollection,
         private $q: ng.IQService,
         private serverSvc: ServerService) {
-        this.rest = rest.withConfig((RestangularConfigurer) => {
-            RestangularConfigurer.setBaseUrl('/api/v1/');
-        });
+        this.rest = rest;
```

Comment:
The way it's set up so far, we're instantiating the resangular for every module - once for normal usage and one for more verbose usage (`restAll`).

This is how we've lifted off the extra config, but I do realize the entire instantiating/rigging has to change eventually. IMO, we should do that once we have tests implemented (so that we don't introduce regressions).

This hasn't been implemented in most efficient way, but it works. So, I got rid of the non-DRY part (`baseUrl` provided as default) and left the rest as is for now.